Updates based on user inputs (#5995)
SAPDocuments/Tutorials: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md

---
title: Create an SAP Fiori App Using SAP Business Application Studio

> description: Develop a simple SAP Fiori application in SAP Business Application Studio. The application will present a list of suppliers from an on-premise backend. The suppliers data will be retrieved from the BusinessPartnerSet collection in the GWSAMPLE_BASIC OData service that is available from SAP's ES5 demo gateway.

Reviewer: Please check grammar.

```suggestion
description: Develop a simple SAP Fiori application in SAP Business Application Studio. The application will present a list of suppliers from an on-premise backend. The suppliers' data will be retrieved from the BusinessPartnerSet collection in the GWSAMPLE_BASIC OData service that is available from SAP's ES5 demo gateway.
```
or

```suggestion
description: Develop a simple SAP Fiori application in SAP Business Application Studio. The application will present a list of suppliers from an on-premise backend. The supplier's data will be retrieved from the BusinessPartnerSet collection in the GWSAMPLE_BASIC OData service that is available from SAP's ES5 demo gateway.
```

[ACCORDION-BEGIN [Step 1: ](Create new SAP Fiori project)]

> 1. In the **Welcome Page** click on **Create project from a template**.

Reviewer: Don't say "click on". either say "click" or pick another word like "choose" or "select".

```suggestion
1. In the **Welcome Page** choose **Create project from a template**.
```

    !![Drag and drop](05-02 AppStudio Layout Editor List Dropped_.jpg)

> 3. Select the **Standard List Item** control by clicking on it (the breadcrumb indicates which control is selected) and, in the **Entity Set** property in the **Properties** pane, click the Bind icon.

Reviewer: ```suggestion
3. Select the **Standard List Item** control by clicking it (the breadcrumb indicates which control is selected) and, in the **Entity Set** property in the **Properties** pane, click the Bind icon.
```
